package.json

```diff
@@ -37,7 +37,7 @@
     "build:ssr": "node build/ssr.js",
     "build:cover": "node build/cover.js",
     "build": "rimraf lib themes/* && run-s build:js build:css build:css:min build:ssr build:cover",
-    "prepare": "npm run build",
+    "prepack": "npm run build",
```

Comment:
+1 for this .

Reply:
As an example, here's another case where prepack was better than prepare: https://github.com/lerna/lerna/issues/2453#issuecomment-586908775

Reply:
What does the release process look like? I see the `pub` script in package.json, but I don't see a `release.sh` file. Where's that file?

That way I can check to make sure the build is ran only once on publish (f.e. we don't want to run `build` and the when we run `publish` it will run `prepack` which will run `build` again).

Reply:
So it turns out that `prepare` is better when using `npm`, and `prepack` is better for Yarn, because Yarn doesn't support `prepare`. So we're not in a good situation because the tools support one or the other.

I am in favor of leaving it as `prepare`, and standardizing the use of `npm` for people that need the build to work when installing from git. Otherwise if we stay with `prepack`, it'll only work with Yarn and NPM users will run into this bug: https://github.com/npm/cli/issues/1229